tests/trainer/test_trainer_tricks.py

```diff
@@ -112,33 +112,34 @@ def test_overfit_batch_limits(tmpdir):
     # ------------------------------------------------------
     for split in (RunningStage.VALIDATING, RunningStage.TESTING):
```

Comment:
can you check for both of the stages here?
for val it should be 0 and for test it shouldn't change at all.

Reply:
Checked, other issues metioned below.